teams.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+set -e
+
+exec env TMPDIR=$XDG_CACHE_HOME \
```

Comment:
You can do `--env=TMDPR=/var/tmp` in finish args to avoid this wrapper script.

Reply:
I prefer to rely on the wrapper script :)

Reply:
Most if not all extra-data applications inline it into manifest.

Reply:
I prefer to keep it separated. More maintainable for me than having it hidden as a bunch of strings inside a json file.

Reply:
Setting `TMDPR` via finish-args would allow you to remove the wrapper altogether.

Reply:
The wrapper script is required for several reasons : 

1. Flatpak will error the build if it can't find the command defined, which will be the case since the binary called in the wrapper is in the extra data.
2. The binary in the extra data is just a shell script that expects to launch `../share/teams/teams`. Due to flatpak using read only filesystems, I can't put their binary inside `/app/bin` and the rest inside `/app/share`.

Also the wrapper script have the advantage to use an env var and to not let the users override this since it is required.

Reply:
>Flatpak will error the build if it can't find the command defined

You can add a symlink (`/app/bin/teams` → `../extra/share/teams/teams` or something).

Reply:
Done.